include/planar_sssp/HenzingerPriorityQueue.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <networkit/graph/Graph.hpp>
+#include <set>
+
+static const int INF = std::numeric_limits<int>::max();
+
+namespace Koala {
+class HPriorityQueue {
```

Comment:
Let's name it HenzingerPriorityQueue - and since it's local to the algorithm, consider hiding it in cpp only - please refer to `koala-networkit\cpp\recognition\PerfectGraphRecognition.cpp` and `perfect` subdirectory for reference.